pkg/mads/v1/generator/meshmetrics.go

```diff
@@ -41,7 +41,7 @@ func Generate(meshMetricToDataplane map[*v1alpha1.Conf]*core_mesh.DataplaneResou
 					Name:        dataplane.GetMeta().GetName(),
 					Address:     net.JoinHostPort(dataplane.GetIP(), strconv.FormatUint(uint64(prometheusEndpoint.Port), 10)),
 					MetricsPath: prometheusEndpoint.Path,
-					Labels:      mads.DataplaneLabels(dataplane, []*core_mesh.MeshGatewayResource{}), // what is this mesh gw and why we have labels there?
+					Labels:      map[string]string{},
```

Comment:
should we just put the comment here that this goes through the meshmetric endpoint?

Reply:
added comment